scripts/quests/outlands/Yomi_Okuri.lua

```diff
@@ -207,16 +207,22 @@ quest.sections =
             {
                 onTrigger = function(player, npc)
                     local vanadielHour = VanadielHour()
+                    local domanMob     = GetMobByID(valkurmID.mob.DOMAN)
+                    local onryoMob     = GetMobByID(valkurmID.mob.ONRYO)
 
                     if
                         player:hasKeyItem(xi.ki.YOMOTSU_HIRASAKA) and
                         quest:getLocalVar(player, 'valkurmNM') == 0 and
-                        (vanadielHour > 18 or vanadielHour < 5) and
-                        not GetMobByID(valkurmID.mob.DOMAN):isSpawned() and
-                        not GetMobByID(valkurmID.mob.ONRYO):isSpawned()
+                        (vanadielHour > 18 or vanadielHour < 5)
```

Comment:
Is this supposed to be 19 to 4 or 18 through 5? (>= and <= or > and <)

Reply:
Definitely should be >=/<=, good catch